internal/pkg/describe/env_test.go

```diff
@@ -152,3 +152,155 @@ func TestEnvDescriber_Describe(t *testing.T) {
 		})
 	}
 }
+
+func TestEnvDescriber_JSONString(t *testing.T) {
```

Comment:
should this be: `TestEnvDescription_JSONString`?

Reply:
Oh, yes! Thanks!